Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/InstanceChangeStreamDelegate.swift

```
@@ -35,7 +35,7 @@ class InstanceChangeStreamDelegate {
             return
         }
 
-        self.namespaceToStreamDelegates[namespace] = NamespaceChangeStreamDelegate(
+        self.namespaceToStreamDelegates[namespace] = try? NamespaceChangeStreamDelegate(
```

[for further discussion] I don't think we should squash errors like this

---

Will do in the error ticket.